bakerydemo/templates/base/include/header-index.html

```diff
@@ -3,8 +3,10 @@
 <div class="container">
     <div class="row">
         <div class="col-md-12">
-            <h1>{{ page.title }}</h1>
-            <p>{{ page.introduction }}</p>
+            <h1 class="index-header__title">{{ page.title }}</h1>
+        </div>
+        <div class="col-sm-12 col-md-7">
+            <p class="index-header__introduction">{{ page.introduction }}</p>
```

Comment:
question: is this a required field in wagtail?

Reply:
Apparently not, added a conditional 👍